conda_mirror/conda_mirror.py

```diff
@@ -505,7 +516,8 @@ def main(upstream_channel, target_directory, temp_directory, platform,
     dict
         Summary of what was removed and what was downloaded.
         keys are:
-        - removed : set of (path, reason) for each package that was removed
+        - validation_results : set of (path, reason) for each package that was validated.
```

Comment:
Is reason a human readable string? A choice from a set of well-known strings (like an enum)?